transport/grpc/dial_enablexds.go

```diff
@@ -1,17 +0,0 @@
-// Copyright 2021 Google LLC.
-// Use of this source code is governed by a BSD-style
-// license that can be found in the LICENSE file.
-
-// Package grpc supports network connections to GRPC servers.
-// This package is not intended for use by end developers. Use the
-// google.golang.org/api/option package to configure API clients.
-
-//go:build enablexds
-// +build enablexds
-
-package grpc
-
-import (
-	// Install google-c2p resolver, which is required for direct path.
-	_ "google.golang.org/grpc/xds/googledirectpath"
```

Comment:
Thanks, please run `go mod tidy`

Reply:
Ah, sorry I forgot this. Just resubmitted. Thanks!